Makefile

```diff
@@ -55,6 +59,16 @@ download.os: $(OS_DOWNLOAD_DIR)/$(FINCH_OS_BASENAME)
 .PHONY: download
 download: download.os
 
+.PHONY: download.lima-dependencies
```

Comment:
nit/suggestion: change this to be more aligned with the OS image download so that user's don't have to redownload the file if there's been no changes. See #90

Reply:
Updated.

Reply:
LGTM, thanks for incorporating that change. It'll make builds faster when user's already have the archives downloaded once!